package/doc/sphinx/source/documentation_pages/topology/TXYZParser.rst

```diff
@@ -1,2 +1,2 @@
 .. automodule:: MDAnalysis.topology.TXYZParser
-
+   :members:
```

Comment:
Add `   :inherited-members:` (although for most formats, we explicitly list the classes in the module itself as this gives finer grained control, e.g., you can specifiy `inherited-members` on a class-by-class basis).